source/common/http/conn_manager_impl.cc

```diff
@@ -416,6 +417,8 @@ ConnectionManagerImpl::ActiveStream::ActiveStream(ConnectionManagerImpl& connect
           connection_manager_.stats_.named_.downstream_rq_time_, connection_manager_.timeSource())),
       stream_info_(connection_manager_.codec_->protocol(), connection_manager_.timeSource()),
       upstream_options_(std::make_shared<Network::Socket::Options>()) {
+  ScopeTrackerImpl scope(this, connection_manager_.read_callbacks_->connection().dispatcher());
```

Comment:
How did you pick the places where you setup the tracked scope? Can we document in comments somewhere? It seems like we should be potentially tracking the encode side as well? Filter continuations? Etc.? I wonder if there is some more generic way we could do this? Nothing immediately comes to mind but I figured I would throw it out there.

Reply:
That's in PR description - I'm going to do encode side (from the router filter) as well as alarms for L7 in follow-ups :-)

Reply:
OK sounds good.